DUI3-DX/Sdk/Speckle.Converters.Common.DependencyInjection/NamedTypeInjector.cs

```diff
@@ -1,24 +1,25 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Reflection;
 using Autofac;
+using Speckle.Converters.Common.Objects;
 using Speckle.Core.Logging;
 
-namespace Speckle.Autofac.DependencyInjection;
+namespace Speckle.Converters.Common.DependencyInjection;
 
-public static class NamedTypeInjector
+public static class ConversionTypesInjector
 {
   public static ContainerBuilder InjectNamedTypes<T>(this ContainerBuilder containerBuilder)
-    where T : class
+    where T : notnull
```

Comment:
not sure about this one, but interesting constraint.

Reply:
oh... on my local I was missing the `where T: class` part, and I was getting warnings as errors in CI requesting `notnull` as constraint.